src/components/form-elements/_form-elements.scss

```diff
@@ -95,30 +93,27 @@ $includeHtml: false !default;
       }
     }
 
-    &:before {
-      $iconSize: $selectFontSize + $iconBoostValue;
-      font-size: $iconSize;
-      font-weight: bold;
+    &__icon {
       position: absolute;
-      right: $selectHeight / 2 - $iconSize / 2;
-      top: 50%;
+      right: ($selectHeight / 4 + rhythm(0.334)) - $selectIconSize / 2;
```

Comment:
Well, its not a variable, its `1/3`. But as soon as we cannot use fraction in a right way - `0.333`